docker/go/Dockerfile

```diff
@@ -6,7 +6,7 @@
 FROM ubuntu:bionic AS builder
 
 ARG PULUMI_VERSION
-ARG LANGUAGE_VERSION
+ARG GO_RUNTIME_VERSION=1.21.1
```

Comment:
We don't need to matrix Go. Go will use the version specified in the go.mod file https://go.dev/doc/toolchain